cmake/catkin_python_setup.cmake

```diff
@@ -163,6 +140,34 @@ function(catkin_python_setup)
       TARGET_NAME ${PROJECT_NAME}_${name}_exec_install
       DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
   endforeach()
+
+  if(${PROJECT_NAME}_SETUP_PY_SETUP_MODULE STREQUAL setuptools)
+    message(STATUS "Appending egg base")
+    set(SETUPTOOLS_EGG_INFO "egg_info --egg-base ${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}")
+  endif()
+
+  assert(PYTHON_INSTALL_DIR)
```

Comment:
Since line 146 already uses the variable this `assert` must be moved above the previous `if`.

Reply:
Resolved in 0c9d05d4